implementations/build-topaz.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e # make script fail on first error
+SCRIPT_PATH=`dirname $0`
+source $SCRIPT_PATH/script.inc
+
+INFO Build Topaz
+pushd $SCRIPT_PATH
+  git clone git://github.com/topazproject/topaz
+  pushd topaz
+    git checkout tim/features
+    pip install invoke
+    export TEST_TYPE=translate-jit-notest
+    invoke travis.install_requirements
+    invoke travis.run_tests
```

Comment:
Really not sure what the error is - translation has finished, I don't get why it is trying to run `run_tests` as command...